server/src/main/java/keywhiz/service/daos/SecretSeriesDAO.java

```diff
@@ -185,6 +184,10 @@ public int setCurrentVersion(long secretId, long secretContentId, String updater
     return Optional.ofNullable(r).map(secretSeriesMapper::map);
   }
 
+  public List<SecretSeries> getMultipleSecretSeriesByName(List<String> names) {
```

Comment:
Probably worth adding a small test for this (the query is simple enough I don't expect there to be a bug, but adding a test is easy).